### DISCUSSION TITLE
API error - Looking to get OpenOutpaint working

### DISCUSSION THREAD

**Discussion by monoinyo** [vladmandic/sdnext#259] [Q&A]
Hi all, finally got Vladmandic working just one last extension giving me problems: https://github.com/zero01101/openOutpaint-webUI-extension

This extension needs API access but when I add it to args I get this: `launch.py: error: ambiguous option: --api could match --api-auth, --api-log
Press any key to continue . . .`

hopefully that's clear, any thoughts?

**Comment by vladmandic** [comment]
this repo works with api always on, you should not add `--api`.
so if extension doesn't work out-of-the-box, create an issue for that.

btw, if you want to enable api auth, those options are still valid.

**Reply by monoinyo** [reply]
Ah I understand, thanks for the quick help! I'll look into starting the issue, I'll see if I can confirm it's not working.
